### PR TITLE
docs: Update monitoring overview

### PR DESCRIPTION
**Pull request by sixhobbits**

### CODE REVIEW

File: astro/src/content/docs/operate/secure-and-monitor/opentelemetry.mdx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+<img src="/img/docs/operate/secure-and-monitor/opentelemetry-display.png" alt="Exploring telemetry using Honeycomb." width="1200" />
+
+### Set Up the Exporter
+
+Download the [Java OpenTelemetry agent](https://github.com/open-telemetry/opentelemetry-java-instrumentation). Make it available to the FusionAuth instance. If you are using a zip/deb/rpm, put it in a directory. If using docker, mount the host directory to the container.
+Set an environment variable `OT_PATH` to point to the directory.
+
+Next, you need to configure FusionAuth to use the agent by passing the correct Java command line switches. There are a couple of ways to do this.
+
+You can use Java system properties, environment variables, or edit the `fusionauth.properties` config file. See the [configuration reference](/docs/reference/configuration) for more info. For this example, environment variables will be used.
```
Comment (alex-fusionauth) [inline astro/src/content/docs/operate/secure-and-monitor/opentelemetry.mdx:22]:
```suggestion
You can use Java system properties, environment variables, or edit the `fusionauth.properties` configuration file. See the [configuration reference](/docs/reference/configuration) for more info. For this example, environment variables will be used.
```

**Review by alex-fusionauth — CHANGES_REQUESTED** [review]
Update Vale `.github/vale/styles/config/vocabularies/FusionAuth/accept.txt`
Grafana
Alertmanager

Is there a better A11y version of the table?

**Comment by sixhobbits** [comment]
> Is there a better A11y version of the table?

@alex-fusionauth -  just to clarify, you mean not using the emoji ✅ etc for the feature comparison?

**Comment by alex-fusionauth** [comment]
> 
> > Is there a better A11y version of the table?
> 
> @alex-fusionauth -  just to clarify, you mean not using the emoji ✅ etc for the feature comparison? 

Yeah exactly

**Comment by sixhobbits** [comment]
> > > Is there a better A11y version of the table?
> > 
> > 
> > @alex-fusionauth -  just to clarify, you mean not using the emoji ✅ etc for the feature comparison?
> 
> Yeah exactly

we could change replace icons for "yes" and "no"?  But I'm not sure it would make that much difference as I think a screen reader would just read out "checkmark" and "restricted symbol" otherwise, and it would make it a bit harder to get a quick overview for people who don't need screenreaders. Or did you have something specific in mind?

**Comment by alex-fusionauth** [comment]
> > > > Is there a better A11y version of the table?
> > > 
> > > 
> > > @alex-fusionauth -  just to clarify, you mean not using the emoji ✅ etc for the feature comparison?
> > 
> > 
> > Yeah exactly
> 
> we could change replace icons for "yes" and "no"? But I'm not sure it would make that much difference as I think a screen reader would just read out "checkmark" and "restricted symbol" otherwise, and it would make it a bit harder to get a quick overview for people who don't need screenreaders. Or did you have something specific in mind?

@sixhobbits we have this table that has "Yes" "No" and "-" for not relevant. I am thinking we should align with this. Even though I like the emoji style better ;)

https://fusionauth.io/docs/lifecycle/authenticate-users/oauth/endpoints#client-secret

**Comment by alex-fusionauth** [comment]
@sixhobbits I think we are probably hitting our own updates here but can you merge main again?

**Comment by alex-fusionauth** [comment]
@sixhobbits conflicts 🙏

**Comment by alex-fusionauth** [comment]
@sixhobbits conflicts 🙏

**Review by alex-fusionauth — APPROVED** [review]

**Pull request merged**